setup_workspace.sh

```diff
@@ -0,0 +1,23 @@
+# This file is used for Bazel CI to install rust and setup the workspace.
+
+# install rust
+apt-get update && apt-get install curl -y && curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+# move workspace file to parent dir
+mv test_suite/WORKSPACE WORKSPACE
```

Comment:
Moving the WORKSPACE file seems brittle. This can break absolute labels in the BUILD files.

You can put the WORKSPACE file at the root of the repository instead (we might need it in the future for other things).